### DISCUSSION TITLE
Cluster subnavigation as iconButtons

### DISCUSSION THREAD

**Discussion by bvicini** [filamentphp/filament#13691] [Help]
### Package

Panel builder

### Package Version

v3.2.96

### How can we help you?

Sometimes, when tables are quite large (many columns) and we cluster resources the space assigned to subnavigation buttons, just to display the resource name, is simply too much. Is it possible to reduce the subnavigation to icons like the main navigation panel? Can be a fixed developer design (per cluster),  not necessarily something the user can expand/compact via a button in the UI. thank you

**Comment by leandrocfe** [comment]
What about this?
![image](https://github.com/user-attachments/assets/7e9e4c91-2d37-422c-a909-378e4b439f05)
This configuration in your Resource will display your resource in a tab

**Reply by bvicini** [reply]
Uhn, my fault! I totally missed to consider subnavigationposition :( Thanks for your time, this can be a valid solution.
